.travis.yml

```diff
@@ -13,24 +13,27 @@ services:
 
 env:
   - BACKEND: 'geonode.geoserver'
-#  FIXME. Will be uncommented when qgis_server backend exists
-#  - BACKEND: 'geonode.qgis_server'
-#    DOCKER_COMPOSE_VERSION: 1.11.2
-#    # This is qgis server address
-#    QGIS_SERVER_URL: http://localhost:9000/
-#    # This is the location of docker network bridge
-#    # So QGIS server can access this address
-#    SITEURL: http://localhost:8000/
-#    GEONODE_PROJECT_PATH: /home/travis/build/kartoza/geonode/
-#    ON_TRAVIS: True
-#    CELERY_ALWAYS_EAGER: True
+  - BACKEND: 'geonode.qgis_server'
+    DOCKER_COMPOSE_VERSION: 1.11.2
+    # This is qgis server address
+    QGIS_SERVER_URL: http://localhost:9000/
+    # This is the location of docker network bridge
+    # So QGIS server can access this address
+    SITEURL: http://localhost:8000/
+    GEONODE_PROJECT_PATH: /home/travis/build/kartoza/geonode/
+    QGIS_SERVER_PORT: 9000
+    ON_TRAVIS: True
+    CELERY_ALWAYS_EAGER: True
 
 branches:
   only:
     - master
     - 2.7.x
     - 2.6.x
 
+before_install:
+  - scripts/misc/qgis_server_setup.sh before_install
```

Comment:
Is it ok if I put some QGIS Server preparation for Travis here (and similar Travis steps)?
Or maybe I have to put it inside paver script?
I'm not sure what is the best way to do this.